### DISCUSSION TITLE
403 API rate limit of 5000 still exceeded

### DISCUSSION THREAD

**Discussion by GopikaV24** [actions/actions-runner-controller#2007] [Questions]
Hi Team,

We have created selfhosted autoscaler using actions-runner-controller in our AKS cluster.  We are using webhook-driven autoscaling option for the autoscaler. But today we got issue like below,

![image](https://user-images.githubusercontent.com/60219629/201093846-23aaf8c9-84e4-400c-9134-bac30f9662cc.png)

Is there any way to bypass this?

**Comment by GopikaV24** [comment]
@mumoshu Can you please provide your suggestion?

**Reply by mumoshu** [reply]
Hey. Perhaps try GitHub App instead of PAT for authentication for more quota?
